handbook/engineering/cloud/index.md

```diff
@@ -6,6 +6,28 @@ The cloud team owns all work that is necessary to build, secure, scale, and oper
 
 1. Any user or organization can use Sourcegraph Cloud with both the private and public code that they care about in a secure way.
 
+## Milestones
+
+1. Any user can add public code
+    - Code is indexed.
+    - It's easy to search over only their code (vs all public code).
```

Comment:
@keegancsmith, @sourcegraph/search: I think this could be a dependency on some substantial search changes. I see two ways in which we can make a user's code easier to find / prioritised:

1. Automatic `repogroups` per user / organization and making that the default search scope.
2. Ranking support in search, which would rank a user's or organization's repos higher (as well as their dependencies).

I think we probably need 1. first, then 2. What do you think? Would you would take these on?

Reply:
We have the concept of version contexts which can likely be extended here. Setting the default scope of a query may be more straight forward now from a code perspective. But if its the right thing to do is another question which is good for search to decide on. I think ideally we have better ranking (might be investigated after streaming).

However, from a code perspective limiting down which repos are searched based on the query is a relatively straightforward change. I think as you are implementing this milestone experimenting with that and seeing how it feels would make a lot of sense for the people who are owning this milestone. This is likely better than potentially getting blocked by search who are working on other projects (potentially).